### ISSUE TITLE
Package not found

### ISSUE THREAD

**Issue by deadcoder0904** [Luehang/react-native-masonry-list#6]
Hey, I'm getting this error -

```js
While trying to resolve module `react-native-masonry-list` from file
`/myApp/src/components/Masonry/Masonry.js`, 
the package `/myApp/node_modules/react-native-masonry-list/package.json` was 
successfully found. However, this package itself specifies a `main` module field
that could not be resolved 
(`/myApp/pix/node_modules/react-native-masonry-list/src/index.js`.
Indeed, none of these files exist.
```

I'm on the latest `^2.7.3` version.

**Comment by deadcoder0904** [comment]
Works fine for `2.7.1` which I'm using now :)

**Comment by Luehang** [comment]
Hi,

Can you reproduce the error?

**Comment by deadcoder0904** [comment]
It's just a simple example like on the README. I copy-pasted nothing else.

**Comment by Luehang** [comment]
On a **clean** React Native environment, it seems to work fine.

**Comment by deadcoder0904** [comment]
For reproducing, you can just go to https://snack.expo.io/ & try importing like this

```js
import MasonryList from "react-native-masonry-list";
```

It shall throw an error `An error occurred when resolving react-native-masonry-list` like


<img width="1430" alt="react-native-masonry-list" src="https://user-images.githubusercontent.com/16436270/53078343-fd33dd80-3519-11e9-9af1-8e49c310dbca.png">

**Comment by deadcoder0904** [comment]
Btw thanks for creating this package. Earlier version works 🎉

**Comment by Luehang** [comment]
Nothing was modified in the package file except version bumping.  I don't think this is a `react-native-masonry-list` problem, if version `2.7.1` works.

**Comment by Luehang** [comment]
Issue was resolved on version `2.7.4`.  There was a publishing issue that didn't include the main files in version `2.7.3`.  Thanks for letting me know!

**Comment by Luehang** [comment]
Can you star this project if you like it?
